pkg/cli/interactive_tests/test_missing_log_output.tcl

```diff
@@ -105,26 +103,3 @@ eexpect "goroutine"
 eexpect ":/# "
 
 end_test
-
-
-
-start_server $argv
-
-start_test "Test that quit does not show INFO by default with --logtostderr"
-# Test quit as non-start command, this time with --logtostderr. Test
```

Comment:
The point of this test was to check the logging behavior of a non-start command, not that of quit specifically.

As such the test is still valuable, you'd just need to use a different command than quit that produces log output.

Reply:
Updated to use `init`.